Bio/Align/__init__.py

```diff
@@ -932,6 +933,82 @@ def sort(self, key=None, reverse=False):
         else:
             self._records.sort(key=key, reverse=reverse)
 
+    @property
+    def substitutions(self):
```

Comment:
Should the gap character be an argument? If it were then this wouldn't work as a property but would have to be a normal method.

Reply:
I expect that in the near future the alignment in the class won't be stored in text form, and once this happens the gap character will lose its meaning. So if we include the gap character as an argument here, we'd have to change the signature of this method once we drop the gap character.

Reply:
And you think an argument-less property will still make sense?

Reply:
Yes; dropping the gap character will only affect the internal representation of the alignment.

Reply:
Good, thanks.